src/librustc_codegen_llvm/lib.rs

```diff
@@ -270,6 +272,47 @@ impl CodegenBackend for LlvmCodegenBackend {
         )
     }
 
+    fn link(
```

Comment:
Please remove the link code from `join_codegen_and_link` and call `link` after `join_codegen` from `rustc_driver`.

Reply:
Created #68601 for this.

Reply:
Done.